js/friday.js

```diff
@@ -1,34 +1,34 @@
-function onLoad() {
-    let today = new Date().getDay();
-    let element = document.getElementById("friday");
-    if (today === 5) {
-      element.innerHTML = "YES";
-      let month = new Date().getMonth();
+(function IsItFridayYet() {
+  const date = new Date();
+  const theMonth = date.getMonth();
+  const theDay = date.getDay();
+  const element = document.getElementById("friday");
+  const spans = document.getElementsByClassName("fallingElement");
+  let season = "";
 
-      let season = "";
-      switch (true) {
-        case month >= 3 && month < 6:
-          season = "spring";
-          break;
-        case month >= 6 && month < 9:
-          season = "summer";
-          break;
-        case month >= 9 && month < 12:
-          season = "autumn";
-          document.body.style.backgroundColor = "fff7c8";
-          break;
-        default:
-          season = "winter";
-          document.body.style.backgroundColor = "dcf0f4";
-          break;
-      }
-      let spans = document.getElementsByClassName("fallingElement");
-      Array.from(spans).forEach(function (element) {
-        element.classList.add(season);
-      });
-    } else {
-      element.innerHTML = "NO";
+  if (theDay === 5) {
```

Comment:
I moved variables to the top. This isn't actually necessary but for the sake of setting the groundwork for being really rigid about how define our inner functions, I want to just pop those to the top. Along with that, I renamed those variables

```
const date = new Date();
const theMonth = date.getMonth();
const theDay = date.getDay();
```

Javascript runs through a compilation phase and execution phase. During compilation, variables get injected into memory. This gives us the ability to play with the allocated space a little. Instead of creating a new date for the month and a new date for the day, we are now defining the date in its own variable, and using that instance of date allocated in memory to provide us both the month and day.

Pretty neat!

The rename of variables I am not too sold on, you dont have to accept that suggestion. My goal is to get you to see how variables can function somewhat like a sentence piece. switch to summer if theMonth is this or that. Set Friday to yes if theDay is equal to 5.